src/servlets/bedtime/index.ts

```diff
@@ -51,37 +51,39 @@ const getTrackMetadata = async (trackId: number, ownerId: number | null): Promis
 const getTracksFromCollection = async (collection: any, ownerUser: any): Promise<TrackResponse[]> => {
 
   const trackIds: number[] = collection.playlist_contents.track_ids.map((t: {time: number, track: number }) => t.track)
-  let tracks = []
+  let tracks: TrackModel[] = []
 
   // Fetch tracks if there are IDs
   if (trackIds.length) {
     const unordredTracks = await getTracks(trackIds)
```

Comment:
Just noticed this typo here

```suggestion
    const unorderedTracks = await getTracks(trackIds)
```